rustls-libssl/src/entry.rs

```diff
@@ -180,6 +180,23 @@ entry! {
                     // not a defined operation in the OpenSSL API
                     0
                 }
+                Ok(SslCtrl::SetChain) => {
```

Comment:
not your fault of course but this sort of ioctl-like "mega" API is unpleasant :nauseated_face:

Reply:
Deeply! Especially changing the reference counting behaviour!